disco_aws_automation/disco_deploy.py

```diff
@@ -559,6 +547,66 @@ def handle_blue_green_ami(self, pipeline_dict, ami, old_group,
             self._disco_elb.delete_elb(hostclass, testing=True)
         return False
 
+    def _create_scaling_schedule(self, pipeline_dict, group_name=None, hostclass=None):
+        """ Create scaling schedules from the pipeline dictionary """
+        desired_size = pipeline_dict.get("desired_size", 1)
+        min_size = pipeline_dict.get("min_size", size_as_minimum_int_or_none(desired_size))
+        max_size = pipeline_dict.get("max_size", 0) or size_as_maximum_int_or_none(desired_size)
+
+        self._disco_aws.create_scaling_schedule(
+            min_size,
+            desired_size,
+            max_size,
+            group_name=group_name,
+            hostclass=hostclass
+        )
+
+    def _generate_final_pipeline(self, pipeline_dict, old_group, ami):
+        """Generate pipeline with sizing for post deployment"""
+        new_config = copy.deepcopy(pipeline_dict)
+        new_config["sequence"] = 1
+        new_config["smoke_test"] = "no"
+        new_config["ami"] = ami.id
+
+        # If there is an already existing ASG, use its sizing. Otherwise, use the pipeline's sizing or a
+        # reasonable default.
+        if old_group:
+            desired_size = old_group.desired_capacity
```

Comment:
still not sure why we're using old ASG vals when creating a new ASG :confused: 
maybe your answer above in my other comment will clarify this